### PR TITLE
Feature/dev env/add dev env

### PR DESCRIPTION
**Pull request by janeirikforland**

### CODE REVIEW

File: src/commands/start-app.ts [hunk 1]
```diff
@@ -1,11 +1,13 @@
 import { Command, flags } from '@oclif/command';
-import * as inquirer from 'inquirer';
+// import * as inquirer from 'inquirer';
```
Comment (eskil0312) [inline src/commands/start-app.ts]: Could this be removed?

File: src/commands/start-app.ts [hunk 2]
```diff
@@ -1,11 +1,13 @@
...
 
-interface IStartAppOptions {
-    apps?: string[];
-    progress?: boolean;
-    production?: boolean;
-}
+import start from '../start-app/scripts/start';
+
+// interface IStartAppOptions {
```
Comment (eskil0312) [inline src/commands/start-app.ts]: Could this be removed?

File: src/commands/start-app.ts [hunk 3]
```diff
@@ -22,26 +24,25 @@ export default class StartApp extends Command {
...
-        questions.push({
-            message: 'Please enter a apps to start. E.g. AppKey1 AppKey2 AppKey3',
-            name: 'apps',
-            type: 'input',
-        });
-    }
-    const answers: any = await inquirer.prompt(questions);
+// const promptForMissingOptions = async (options: IStartAppOptions): Promise<object> => {
+//     const questions = [];
+//     if (!options.apps) {
```
Comment (eskil0312) [inline src/commands/start-app.ts]: Could this be removed?

File: src/start-app/scripts/start.ts [hunk 4]
```diff
@@ -1,47 +1,49 @@
...
+    https
+        .createServer(
+            {
+                cert: fs.readFileSync(path.resolve(__dirname, '..', 'keys', 'server.cert')),
+                key: fs.readFileSync(path.resolve(__dirname, '..', 'keys', 'server.key')),
+            },
+            app
+        )
+        .listen(3000, () => {
+            // tslint:disable-next-line:no-console
```
Comment (eskil0312) [inline src/start-app/scripts/start.ts]: Is this necessary when no-console is set to false in tslint.json?

Comment (janeirikforland) [reply src/start-app/scripts/start.ts]: Nope. Removed! :)

File: src/commands/start-app.ts (hunk 3 above)
Comment (janeirikforland) [reply src/commands/start-app.ts]: Yes, but not yet.

File: src/commands/start-app.ts (hunk 2 above)
Comment (janeirikforland) [reply src/commands/start-app.ts]: Yes, but not yet.

File: src/commands/start-app.ts (hunk 1 above)
Comment (janeirikforland) [reply src/commands/start-app.ts]: Yes, but not yet.

**Pull request merged**